lib/sensors/src/URM04Sensor.cpp

```diff
@@ -0,0 +1,211 @@
+#include "URM04Sensor.h"
+
+// instantiate pin connected to the URM04 URM04Sensor
+URM04Sensor::URM04Sensor::URM04Sensor(PinName trig_pin, PinName _RX, PinName _TX, uint8_t default_address)
+    : m_trigPin(trig_pin), startAddr(default_address), RX(_RX), TX(_TX), serial(TX, RX, BAUD_RATE) {
+  // instantiate command buffer with all zeros
+
+  memset(&cmdst[0], 0, sizeof(cmdst));
```

Comment:
I think you don't need to do this as long you make sure to update the cmdst array each time you send a command